sycl/source/CMakeLists.txt

```diff
@@ -112,7 +113,7 @@ set(SYCL_SOURCES
     "detail/builtins_integer.cpp"
     "detail/builtins_math.cpp"
     "detail/builtins_relational.cpp"
-    "detail/pi.cpp"
+    "${pi_source_dir}/pi.cpp"
```

Comment:
Do I understand correctly that building libpi as a separate library will be added in follow-up patches?